3_visualize/src/plot_site_data.R

```diff
@@ -7,7 +7,7 @@ plot_site_data <- function(out_file, site_data, parameter) {
     geom_line() +
     geom_point(data=filter(site_data, !(Quality %in% c('A','P'))), size=0.1) +
     ylab(dataRetrieval::parameterCdFile %>% filter(parameter_cd == parameter) %>% pull(parameter_nm)) +
-    ggtitle(site_data$Site[1])
+    ggtitle(paste(site_data$State, site_data$Site[1]))
```

Comment:
you might need `site_data$State[1]` here, as you have for the site number since it is a data frame with some non-unique values in those columns